master/getting-started/kubernetes/installation/hosted/index.md

```diff
@@ -21,7 +21,7 @@ the recommended hosted approach for deploying Calico in production.
 This manifest installs Calico as well as a single node etcd cluster.  This is the recommended hosted approach
 for getting started quickly with Calico in conjunction with tools like kubeadm.
 
-#### [Etcdless Hosted Install](k8s-backend/)
+#### [Kubernetes Datastore Driver](k8s-backend/)
```

Comment:
I think this should read `Kubernetes Datastore Hosted Install`

(and other references to "Kubernetes datastore driver" should probably just be "Kubernetes datastore")